execution-core/src/transfer/contract_exec.rs

```diff
@@ -21,11 +21,14 @@ use crate::{ContractId, Error, ARGBUF_LEN};
 /// Data for either contract call or contract deployment.
 #[derive(Debug, Clone, PartialEq, Eq, Archive, Serialize, Deserialize)]
 #[archive_attr(derive(CheckBytes))]
+#[allow(clippy::large_enum_variant)]
 pub enum ContractExec {
     /// Data for a contract call.
     Call(ContractCall),
     /// Data for a contract deployment.
     Deploy(ContractDeploy),
+    /// Data used by some exchanges to differentiate target accounts.
```

Comment:
I would avoid to mention exchanges as the only one who are using this memo

Reply:
You're right. Will change the comment.